ffmpeg_quality_metrics/__main__.py

```diff
@@ -29,6 +29,12 @@
 NUL = 'NUL' if IS_WIN else '/dev/null'
 
 
+def win_path_check(path):
+    if IS_WIN:
+        return path.replace("\\", "/").replace(":", "\\\\:")
+    return path
```

Comment:
can change to `return path if not IS_WIN else path.replace("\\", "/").replace(":", "\\\\:")` if you want it as one liner